src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,296 @@
+            "|_______||_______||______| |_______|  |___|  |__| |__||_|   |_||__| |__|\n";
+    private static final String SEPARATOR = "zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz\n";
+    private static final int MAX_TASK = 100;
+    private static ArrayList<Task> userInputs = new ArrayList<>();
+    private static int userInputsCount = 0;
+
+    /**
+     * Initiate the program and print the various messages
+     */
+    public static void initiateDuke() {
+        System.out.println(SEPARATOR + "Hi... from GUDETAMA... so sleepy\n" + LOGO);
+        System.out.println("Give me five more minutes..... What can I do for you?\n" + SEPARATOR);
+
+        try {
+            loadSave();
+        } catch (FileNotFoundException e) {
+            System.out.println("File not found" + System.lineSeparator() + "Empty file created");
+            System.out.println(SEPARATOR);
+        }
+    }
+
+    public static void loadSave() throws FileNotFoundException {
+        final String LOAD_DELIMITER = "--";
+        final int TASK_INDEX = 0;
+        final int DONE_INDEX = 1;
+        final int DESCRIPTION_INDEX = 2;
+        final int BY_AT_INDEX = 3;
+        final String TODO_CODE = "T";
+        final String DEADLINE_CODE = "D";
+        final String EVENT_CODE = "E";
+        String line;
+        File dir = new File("data");
+
+        if (!dir.exists()) {
+            dir.mkdir();
+        }
+
+        File f = new File("data/duke.txt");
+        Scanner s = new Scanner(f);
+
+        while (s.hasNext()) {
+            line = s.nextLine();
+            String[] lineData = line.split(LOAD_DELIMITER);
+
+            if (lineData[TASK_INDEX].equals(TODO_CODE)) {
+                userInputs.add(new Todo(lineData[DESCRIPTION_INDEX]));
+            } else if (lineData[TASK_INDEX].equals(DEADLINE_CODE)) {
+                userInputs.add(new Deadline(lineData[DESCRIPTION_INDEX], lineData[BY_AT_INDEX]));
+            } else if (lineData[TASK_INDEX].equals(EVENT_CODE)) {
+                userInputs.add(new Event(lineData[DESCRIPTION_INDEX], lineData[BY_AT_INDEX]));
+            }
+
+            if (lineData[DONE_INDEX].equals("1")) {
+                userInputs.get(userInputsCount).markAsDone();
+            }
+            userInputsCount++;
+        }
+    }
+
+    /**
+     * Prompt for user inputs
+     */
+    public static void promptInput() {
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+
+        while (!line.equals("bye")) {
+            processInput(line);
+            line = in.nextLine();
+        }
+
+        try {
+            printEnd();
+        } catch (IOException e) {
+            System.out.println("Something went wrong: " + e);
+        }
+    }
+
+    /**
+     * Process user input
+     * @param input the user input
+     */
+    public static void processInput(String input) {
+        input = input.trim();
+
+        try {
+            if (input.startsWith("list")) {
```

Comment:
Perhaps these magic strings could be declared as constants? Also, the methods could perhaps belong to another class?